### PR TITLE
[Images] Added Images batch API content

### PR DESCRIPTION
**Pull request by dcpena**
Added Images batch API content to address PCX-8139.

### CODE REVIEW

File: content/images/cloudflare-images/upload-images/images-batch.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+# Images batch API
+
+The Images batch API lets you make several requests in sequence while bypassing Cloudflare’s global API rate limits. 
+
+To use the Images batch API, you will need to obtain a batch token and use the token to make several requests. The requests authorized by this batch token are made to a separate endpoint and do not count toward the global API rate limits.
+
+
+To obtain a token, you can use the new `images/v1/batch_token` endpoint as shown in the example below.
```
Comment (marciocloudflare) [inline content/images/cloudflare-images/upload-images/images-batch.md:12]:
```suggestion
To obtain a token, you can use the new `images/v1/batch_token` endpoint as shown in the example below.
```

File: content/images/cloudflare-images/upload-images/images-batch.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+{
+  "result": {
+    "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiIsImtpZCI6ImJhdGNoIn0.eyJzdWIiOiJWaTd3aTVLU0l0eEdGc1dSRzJVczZRIiwiZXhwIjoxNjkxNTk1MjM2fQ.qF8Yx21ohxuYf6RLXvLmX6dYhwVQ81bSeaQQOpwQEC0",
+    "expiresAt": "2023-08-09T15:33:56.273411222Z"
+  },
+  "success": true,
+  "errors": [],
+  "messages": []
+}
+
```
Comment (marciocloudflare) [inline content/images/cloudflare-images/upload-images/images-batch.md]:
```suggestion
```

File: content/images/cloudflare-images/upload-images/images-batch.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+To obtain a token, you can use the new `images/v1/batch_token` endpoint as shown in the example below.
+
+```bash
+$ curl -H "Authorization: Bearer <CLOUDFLARE_API_TOKEN>" \
+  "https://api.cloudflare.com/client/v4/accounts/ACCOUNT_ID/images/v1/batch_token"
+
+{
+  "result": {
+    "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiIsImtpZCI6ImJhdGNoIn0.eyJzdWIiOiJWaTd3aTVLU0l0eEdGc1dSRzJVczZRIiwiZXhwIjoxNjkxNTk1MjM2fQ.qF8Yx21ohxuYf6RLXvLmX6dYhwVQ81bSeaQQOpwQEC0",
```
Comment (marciocloudflare) [inline content/images/cloudflare-images/upload-images/images-batch.md]: Should we show this?

**Review by marciocloudflare — APPROVED** [review]

File: content/images/cloudflare-images/upload-images/images-batch.md (hunk 3 above)
Comment (dcpena) [reply content/images/cloudflare-images/upload-images/images-batch.md]: Good callout. I had the same thought. Let me double-check!

**Pull request merged**